app/assets/javascripts/utilities/getCsrfToken.js

```diff
@@ -1,5 +1,8 @@
 /* global Honeybadger */
 
+const MAX_RETRIES = 30;
```

Comment:
Just seeing this back from vaction, but I had a thought. I wonder if we could use a MutationObserver listening for changes in the `<head />` @aitchiss ?

Reply:
Certainly could be a bit more efficient - we'd still need a timeout, but it could be that after a fixed amount of time we cancel the observer and return the error 🤔